tests/test_cli/test_utils/test_config.py

```diff
@@ -60,3 +64,24 @@ def test_validate_item_config_negative(self, *mocks):
         """Test validate_item_config for negative result."""
         with self.assertRaises(AEAConfigException):
             validate_item_config(item_type="agent", package_path="file/path")
+
+
+def test_config_validator() -> None:
+    """Test config schema validation."""
+
+    config = DEFAULT_CLI_CONFIG.copy()
+    validate_cli_config(config)
+
+    with pytest.raises(
+        jsonschema.exceptions.ValidationError,
+        match=re.escape("None is not of type 'string'"),
+    ):
+        config["author"] = None  # type: ignore
+        validate_cli_config(config)
+
+    with pytest.raises(
+        jsonschema.exceptions.ValidationError,
+        match=re.escape("'author' is a required property"),
+    ):
```

Comment:
if we do not allow missing author (or name) let's also remove the empty string default values here?

https://github.com/valory-xyz/open-aea/blob/85b1a6f8e56ad675a6ea2393a7e4a27f86e8b19a/aea/cli/registry/push.py#L65-L66

Reply:
That's a component config, we're adding validation for CLI config.